sdcm/fill_db_data.py

```diff
@@ -804,7 +804,6 @@ class FillDatabaseData(ClusterTester):
             'invalid_queries': [
                 "INSERT INTO null_support_test_old_version (k, c, v2) VALUES (0, 2, {1, null})",
                 "INSERT INTO null_support_test_old_version (k, c, v2) VALUES (0, 0, { 'foo', 'bar', null })",
-                "SELECT * FROM null_support_test_old_version WHERE k = null",
```

Comment:
null_support_test will be executed for new scylla with the support.
null_support_test_old_version will be executed for new scylla without the support.
- 2020.1, 2021.1

So the query won't effect anything in upgrade test of scylla-master

But we still need it in those upgrade tests:
- upgrade from 2021.1 to 2022.1
- upgrade from 2020.1 to 2021.1

Let's revert this PR and re-add the query back to null_support_test_old_version test?

/Cc @vponomaryov @roydahan

Reply:
Agree with Amos.
I don't see any need in this change, since the updated set of queries is related only to old Scylla versions, not new ones where it is already absent.
So, this change is just reduces number of checks on old versions, no more, no less.